### PR TITLE
Added Piratebox to Groupware/File Sharing and Sync

### PR DESCRIPTION
**Pull request by edouerd**
Group-centric file sharing and sync hardware/software system — http://piratebox.cc/start

### CODE REVIEW

**Comment by Kickball** [comment]
Would you mind adding a link to the source code (https://github.com/PirateBox-Dev)?

Also this is quite a unique item, as it has a hardware aspect. What do you think @nodiscc?

**Comment by edouerd** [comment]
@Kickball 

Edited the item's URL from `http://piratebox.cc/` to `https://github.com/PirateBox-Dev`

**Comment by edouerd** [comment]
Also, unique in the sense that it may not necessarily belong in the list? Or unique in that it might require a new section?

**Comment by Kickball** [comment]
Sorry firstly, can I ask you to put ``http://piratebox.cc/`` back as the main link and add a source code closer to the end of the line.

I was thinking that it was unique and might not belong on the list.

**Comment by edouerd** [comment]
@Kickball Fixed :+1:

**Comment by nodiscc** [comment]
@edouerd Can you move thsi to the [Self-hosting Solutions](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#self-hosting-solutions) section?

**Comment by edouerd** [comment]
@nodiscc Done! :clap:

File: README.md [hunk 1]
```diff
@@ -566,6 +566,7 @@ See [staticsitegenerators.net](https://staticsitegenerators.net) and [staticgen.
   * [Sandstorm](https://sandstorm.io/) - Personal server for running self-hosted apps easily and securely. ([Demo](https://demo.sandstorm.io/), [Source code](https://github.com/sandstorm-io/sandstorm)) `Apache` `C++/Other`
   * [sovereign](https://github.com/sovereign/sovereign) -  A set of Ansible playbooks to build and maintain your own private cloud: email, calendar, contacts, file sync, IRC bouncer, VPN, and more. `GPLv3` `YAML/Other`
   * [yunohost](https://yunohost.org/) - a server operating system aiming to make self-hosting accessible to everyone. ([Source code](https://github.com/YunoHost)) - `AGPL` `Python/Other`
+  * [Piratebox](http://piratebox.cc/start) - A DIY anonymous offline file-sharing and communications system built with free software and inexpensive off-the-shelf hardware. [Source Code Here](https://github.com/PirateBox-Dev). `GPLv3`
```
Comment (nodiscc) [inline README.md]: Please use the format 

    ([Source code](https://github.com/PirateBox-Dev))

**Comment by edouerd** [comment]
Added `Python/Other`, corrected formatting for source code link inclusion

File: README.md [hunk 2]
```diff
@@ -566,6 +566,7 @@ See [staticsitegenerators.net](https://staticsitegenerators.net) and [staticgen.
   * [Sandstorm](https://sandstorm.io/) - Personal server for running self-hosted apps easily and securely. ([Demo](https://demo.sandstorm.io/), [Source code](https://github.com/sandstorm-io/sandstorm)) `Apache` `C++/Other`
   * [sovereign](https://github.com/sovereign/sovereign) -  A set of Ansible playbooks to build and maintain your own private cloud: email, calendar, contacts, file sync, IRC bouncer, VPN, and more. `GPLv3` `YAML/Other`
   * [yunohost](https://yunohost.org/) - a server operating system aiming to make self-hosting accessible to everyone. ([Source code](https://github.com/YunoHost)) - `AGPL` `Python/Other`
+  * [Piratebox](http://piratebox.cc/start) - A DIY anonymous offline file-sharing and communications system built with free software and inexpensive off-the-shelf hardware. ([Source code](https://github.com/PirateBox-Dev)). `GPLv3` `Python/Other`
```
Comment (nodiscc) [inline README.md]: Please order the entries alphabetically

**Comment by edouerd** [comment]
Sorted properly, anything else?

**Comment by nodiscc** [comment]
I'd appreciate if you could squash your commits into one.

**Comment by edouerd** [comment]
Apologies, will be better about this next time! :+1:

**Pull request merged**